internal/selfupdate/download.go

```diff
@@ -74,33 +67,45 @@ func extractToFile(buf []byte, filename, target string, printf func(string, ...i
 		rd = file
 	}
 
-	err = os.Remove(target)
-	if os.IsNotExist(err) {
-		err = nil
-	}
+	// Write everything to a temp file
+	dir := filepath.Dir(target)
+	new, err := ioutil.TempFile(dir, "restic")
```

Comment:
Since #3680 there's no need to maintain compatibility with Go 1.14. Can you convert this back to use os.CreateTemp ?

Reply:
Haven't seen the discussion above. -> Ignore this comment.